src/components/TwoLevelSelector.js

```diff
@@ -0,0 +1,377 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const StyleConstants = require('../constants/Style');
+
+const TwoLevelSelector = React.createClass({
+
+  propTypes: {
+    data: React.PropTypes.object,
+    onChange: React.PropTypes.func,
+    selectedItem: React.PropTypes.string,
```

Comment:
So, as  a user of this component, I might want more than just a string back.  Sometimes I care more about the associated id or guid of a selected item rather than it's display name.  Can we change this and the code around it to pass back more than a string.  This might require changing how the data prop is formatted but that might make this component a little more usable. 

I imagine the data prop looking like this

```javascript
[
  {
    name: 'groceries'
    id: 1232341234
    subs: [
      {
        //same layout as parent
      }
    ]
  }
]
```